src/jit/codegenxarch.cpp

```diff
@@ -7890,7 +7890,89 @@ CodeGen::getBaseVarForPutArgStk(GenTreePtr treeNode)
     {
         // The first baseVarNum is guaranteed to be the first incoming arg of the method being compiled.
         // See lvaInitTypeRef() for the order in which lvaTable entries are initialized.
+        // For Windows the value is always 0 (there are always slots in the caller's frame for the first 4 args to a callee).
+        // For System V the first stack arg should be calculated.
         baseVarNum = 0;
+
```

Comment:
Good suggestion, though I would call it something like getFirstArgWithStackLocation(), since otherwise it would be confusing that on windows it returns an arg that's passed in a register.